sdk/react/controlBar.js

```diff
@@ -136,15 +137,18 @@ var ControlBar = React.createClass({
         durationStyle: [styles.durationLabel, {"fontSize": labelFontSize}],
         completeTimeStyle: [styles.completeTimeStyle],
         playHeadTimeString: this.getPlayHeadTimeString(),
+        iconTouchableStyle: styles.iconTouchable,
         durationString: this.getDurationString()
       },
       fullscreen: {
         onPress: this.onFullscreenPress,
+        iconTouchableStyle: styles.iconTouchable,
         style: [styles.icon, {"fontSize": iconFontSize}, this.props.config.controlBar.iconStyle.active],
         icon: this.props.fullscreen ? this.props.config.icons.compress : this.props.config.icons.expand
       },
       moreOptions: {
         onPress: this.onMorePress,
+        iconTouchableStyle: styles.iconTouchable,
```

Comment:
please add this to all of the other buttons (discovery, share, closedCaption, playPause)

If you want to test it, you can add the buttons to the skin.json